### PR TITLE
PAYEE-210 - Compiling - Added in new library, main template and helpers

### PR DESCRIPTION
**Pull request by brendanstraughan**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -53,6 +53,8 @@ feature.ur-banner.enabled = false
 ur-banner.url = "https://www.surveygizmo.com/s3/4856606/HMRC-Letter-Study-7"
 http-verbs.retries.ssl-engine-closed-already.enabled = true
 
+contact-frontend.serviceId = "TES"
```
Comment (brendanstraughan) [inline conf/application.conf:56]: where is this used?

File: app/controllers/benefits/RemoveCompanyBenefitController.scala [hunk 2]
```diff
@@ -204,7 +204,8 @@ class RemoveCompanyBenefitController @Inject()(
           Redirect(controllers.routes.TaxAccountSummaryController.onPageLoad())
         case Right((mandatoryJourneyValues, optionalSeq)) =>
           val stopDate = {
-            val startOfTaxYear = langUtils.Dates.formatDate(TaxYear().start)
+            val startOfTaxYear = "2022"
+            //TODO:This does not compile when we add the new library in - langUtils.Dates.formatDate(TaxYear().start)
```
Comment (brendanstraughan) [inline app/controllers/benefits/RemoveCompanyBenefitController.scala:208]: Will approve when this is fixed by merging in https://github.com/hmrc/tai-frontend/pull/863

File: conf/application.conf (hunk 1 above)
Comment (masahmad) [reply conf/application.conf:56]: required for this helper.....
https://github.com/hmrc/play-frontend-hmrc#helping-users-report-technical-issues

File: project/AppDependencies.scala [hunk 3]
```diff
@@ -19,7 +19,8 @@ private object AppDependencies {
     "uk.gov.hmrc"       %% "play-ui"                          % s"9.7.0-$playVersion",
     "org.typelevel"     %% "cats-core"                        % "2.0.0",
     "com.typesafe.play" %% "play-json-joda"                   % "2.6.10",
-    "uk.gov.hmrc"       %% "digital-engagement-platform-chat" % s"0.18.0-$playVersion"
+    "uk.gov.hmrc"       %% "digital-engagement-platform-chat" % s"0.18.0-$playVersion",
+    "uk.gov.hmrc"       %% "play-frontend-hmrc"               % s"3.0.0-$playVersion"
```
[the remark this answers is not in the document]
Comment (masahmad) [reply project/AppDependencies.scala]: yes updated

**Review by Jamie22Butler — COMMENTED** [review]
LGTM 👍

**Pull request merged**